templates/ts-apollo-mongodb-backend/package.json

```diff
@@ -30,7 +30,7 @@
     "@types/glob": "7.1.3",
     "@types/graphql": "14.2.3",
     "@types/node-fetch": "2.5.7",
-    "graphback-cli": "0.15.1",
+    "graphback-cli": "0.16.0-beta3",
```

Comment:
Should this stay as was or changed or the latest release? [0.16.0-beta4](https://github.com/aerogear/graphback/releases/tag/0.16.0-beta4)?